raiden/tests/integration/rpc/test_assumptions.py

```diff
@@ -61,22 +52,36 @@ def test_call_invalid_selector(deploy_client, blockchain_backend):
     selector = decode_hex(contract_proxy.encode_function_call('ret', args=[]))
     next_byte = chr(selector[0] + 1).encode()
     wrong_selector = next_byte + selector[1:]
-    result = deploy_client.eth_call(
-        sender=deploy_client.sender,
-        to=address,
-        data=wrong_selector,
-    )
+    result = deploy_client.web3.eth.call({
+        'from': to_checksum_address(deploy_client.sender),
+        'to': to_checksum_address(address),
+        'data': wrong_selector,
+    })
     assert result == b''
 
 
+def test_call_inexisting_address(deploy_client, blockchain_backend):
+    """ A JSON RPC call to an inexisting address returns the empty string. """
+
+    inexisting_address = b'\x01\x02\x03\x04\x05' * 4
+
+    assert len(deploy_client.web3.eth.getCode(to_checksum_address(inexisting_address))) == 0
+    transaction = {
+        'from': to_checksum_address(deploy_client.sender),
+        'to': to_checksum_address(inexisting_address),
+        'data': b'',
+        'value': 0
+    }
+    assert deploy_client.web3.eth.call(transaction) == b''
+
+
 def test_call_throws(deploy_client, blockchain_backend):
     """ A JSON RPC call to a function that throws returns the empty string. """
     contract_proxy = deploy_rpc_test_contract(deploy_client)
 
     address = contract_proxy.contract_address
     assert len(deploy_client.web3.eth.getCode(to_checksum_address(address))) > 0
-
-    assert contract_proxy.call('fail') == b''
+    assert contract_proxy.contract.functions.fail().call() == []
```

Comment:
This is one of the reasons for having these assumptions tests, the return type changed from an empty binary string to an empty list.

Raiden has a few places that it's assumed the empty string is used to represent a throw, e..g:

https://github.com/raiden-network/raiden/blob/745a85b26604283c436f53083e306dbe8a77b4d1/raiden/network/proxies/registry.py#L80-L82

I believe that for now we are good, since these proxies will be replaced very soon.